vmdb/product/toolbars/ems_infra_center_tb.yaml

```diff
@@ -22,6 +22,11 @@
       :text: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :title: 'Edit this #{ui_lookup(:table=>"ems_infra")}'
       :url: '/edit'
+    - :button: ems_infra_scale
```

Comment:
hm does this mean the button will be visible in all providers, or is that somehow handled with rbac? Not sure about the other ymls.

@martinpovolny ^ do you know on what condition buttons are shown?